backbone/src/main/res/values-ja-rJP/strings.xml

```diff
@@ -160,4 +160,5 @@
     <string name="rsb_edit_step_alert_step_skip_content">[ja-rJP] Your original response will be discarded..</string>
     <string name="rsb_edit_step_alert_step_skip_discard">[ja-rJP] NO, DON\'T SKIP</string>
     <string name="rsb_edit_step_alert_step_skip_positive">[ja-rJP] Yes, skip</string>
+    <string name="rsb_edit_step">[ja-rJP] Edit</string>
```

Comment:
Maybe it's the same issue for all these strings but why not [ja-JP] ?

Reply:
I'm just using the file name postfix `values-ja-rJP`

Reply:
I understand but for all the other when it's [fr-rFR] we add the prefix [fr-FR] not [fr-rFR] ^^ But i think it's not really important